themes/mongodb/src/js/componentTabs.js

```diff
@@ -12,12 +12,12 @@ function showHideTabContent(currentAttrValue) {
 class TabsSingleton {
     constructor(key) {
         this.key = key;
-        this.tabStrip = document.querySelector('.tab-strip--singleton');
+        this.tabStrips = document.querySelectorAll('.tab-strip--singleton');
 
         // Only tab sets will have a type, init and try to retrieve
         this.type = null;
-        if (this.tabStrip !== null) {
-            this.type = this.tabStrip.getAttribute('data-tab-preference');
+        if (this.tabStrips !== null) {
```

Comment:
`this.tabStrips` will never be `null` given the above switch to `querySelectorAll`.

This should read `if (this.tabStrips.length > 0) {`